DemoViperTests/Modules/ListReposSpec/ReposInteractorSpec.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ReposInteractorSpec.swift
+//  DemoViper Tests
+//
+//  Created by Nguyen M. Tam on 4/20/20.
+//  Copyright © 2020 Nimble Co. Ltd. All rights reserved.
+//
+
+import Quick
+import Nimble
+
+@testable import DemoViper
+
+final class ReposInteractorSpec: QuickSpec {
+
+    override func spec() { 
+
+        var interactor: ReposInteractor!
+        var output: ReposInteractorOutputMock!
+        var service: RepoNetworkProtocolMock!
+        let repo1 = Repo(from: """
```

Comment:
Please create a dummy file and move all Repo instances into that, you are not using these instances in 1 place.